src/appleseed/renderer/kernel/texturing/texturestore.cpp

```diff
@@ -77,6 +78,19 @@ StatisticsVector TextureStore::get_statistics() const
     return StatisticsVector::make("texture store statistics", stats);
 }
 
+Dictionary TextureStore::get_params_metadata()
+{
+    Dictionary metadata;
+    metadata.dictionaries().insert(
+        "max_size",
```

Comment:
I think that metadata should reflect the units and values of the parameters as used in the code, 
and it would be the plugin responsibility to expose some parameters in a different way if that's more convenient.

Or do you mean switching max_size everywhere in the code to MBs?